packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -29,11 +29,14 @@ export const useBreadcrumbs = props => {
         variables: { category_id: categoryId }
     });
 
+    const categoryUrlSuffix =
```

Comment:
This default value seems unreachable/unused, do we need it?

Reply:
If `category_url_suffix` is undefined wouldn't we want a fallback?

Reply:
The return type is `String`, so that shouldn't be possible. I don't think we need to guard against GraphQL returning partial results, since we're using the default error handling policy of `none`.

> none: This is the default policy to match how Apollo Client 1.0 worked. Any GraphQL Errors are treated the same as network errors and any data is ignored from the response.